src/test/pom-parser/pom-parser.test.js

```diff
@@ -73,4 +97,111 @@ describe('POMParser', () => {
         });
     });
   });
+
+  it('Child group id inheritance', () => {
+    const rawContent = readFile('child-group-id-inheritance.xml');
+
+    return pomParser.parsePOMFromString(rawContent)
+      .then((pomContent) => {
+          assert.equal(pomContent.groupId, 'a');
+      });
+  });
+
+  describe('TestRepoValues-parsePomFromString', () => {
```

Comment:
So, to me, this doesn't really describe the grouping of tests that you're doing.  Here is what the build output looks like based on the naming that you chose:
![image](https://user-images.githubusercontent.com/37994541/41427419-81030ca8-6fcc-11e8-8e4f-8908e7b2c7a1.png)

We could probably switch around this structure a bit to be something more along the lines of:
```plaintext
- Pom Parser
 - parsePomFromString
    - Parent Coordinates
        - <tests here related to parent pom coordinates>
    - Inheritance From Parent Pom
        - <tests here related to parent pom and inheritance>
    - Project Coordinates
        - <tests here related to getting project coordinates>
    - Project Dependencies
        - <tests here related to dependencies>
    - Project Repositories
        - <tests here related to repository tag for the project>
```